index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
Sorry, that was poorly phrased.

Basically, I'm thinking of changing the PR so that this can be defined externally (i.e. an option passed into sendNotification), like it was before and for the tests, I'll manually check Firefox or chrome and set the audience accordingly.

This does open the question of how should a developer find out what the correct audience is for a push service? Let's say a new browser Y started support push, the audience should be....?